### DISCUSSION TITLE
应该如何声明我使用了这个库?

### DISCUSSION THREAD

**Discussion by iKexinLL** [dotnetcore/BootstrapBlazor#2566] [Q&A] [answered]
在"关于"页面做个引用吗?比如:

[此软件使用了BootstrapBlazor开源库](https://gitee.com/LongbowEnterprise/BootstrapBlazor)

**Comment by densen2014** [comment]
MIT协议，你可以自由使用

**Comment by ArgoZhang** [comment]
@iKexinLL 那个关于？可以截图吗？

**Reply by iKexinLL** [reply]
类似这个样子
![11223](https://github.com/dotnetcore/BootstrapBlazor/assets/13914319/435c112b-c947-4338-a502-d09f2d27e192)

**Comment by ArgoZhang** [comment, marked as answer]
@iKexinLL 这个可以的。也可以放到一个独立的页面里。不声明也可以的，咱们这个库是 `Apache` 开源协议，非常宽松

**Reply by iKexinLL** [reply]
了解. 为宣传尽一份力量

**Reply by ArgoZhang** [reply]
@iKexinLL 感谢感谢大佬了
